### ISSUE TITLE
cylc warnings: started before submitted

### ISSUE THREAD

**Issue by benfitzpatrick** [cylc/cylc#905]
Some tasks regularly receive a started message before the submitted
message can get through. This causes the following message to
appear in the suite error file:
```
2014/02/13 15:53:04 WARNING - [foo.2014021312] -Assuming non-reported outputs were completed:
foo.2014021312 submitted
```
This can be the dominant warning or error message in suites.

We should suppress this message somehow.

**Comment by matthewrmshin** [comment]
@benfitzpatrick Are we still seeing this? Or has this been fixed by #1012 (i.e. the son of #891)?

**Comment by benfitzpatrick** [comment]
I can't see it in any cylc 6 logs. Very happy to see it gone!
